crates/revm/revm-inspectors/src/tracing/builder/geth.rs

```diff
@@ -147,4 +149,70 @@ impl GethTraceBuilder {
             }
         }
     }
+
+    pub fn geth_prestate_traces<DB>(
+        &self,
+        ResultAndState { state, .. }: &ResultAndState,
+        PreStateConfig { diff_mode }: PreStateConfig,
+        db: DB,
+    ) -> Result<PreStateFrame, DB::Error>
+        where DB: DatabaseRef {
+        match diff_mode {
+            Some(_) => {
```

Comment:
we want this to be obtainable via a new function on `PreStateConfig` because here we still have `Some(false)`

so we need a function that returns a bool via unwrap_or_default